packages/core/src/external/fhir/consolidated/deduplicate.ts

```diff
@@ -22,14 +26,13 @@ export async function deduplicate({
     distinctId: cxId,
     event: EventTypes.fhirDeduplication,
     properties: {
-      patientId: patientId,
-      initialBundleLength: bundle.entry?.length,
-      finalBundleLength: dedupedBundle.entry?.length,
-      duration,
+      patientId,
+      preDedupBundleSize: bundle.entry?.length,
+      postDedupBundleSize: dedupedBundle.entry?.length,
+      dedupDurationMs: duration,
```

Comment:
If we have a function that calculates duration based on a `startedAt` timestamp, and maybe something that we could store the context (cxId, patientId), these calls would be much simpler.

Also cool to just ticket as improvemenet for Posthog/analytics.